behaviour/graql/language/delete.feature

```diff
@@ -785,7 +1308,7 @@ Feature: Graql Delete Query
     Then answer size is: 0
 
 
-  Scenario: delete key ownership throws exception
+  Scenario: deleting a key ownership throws an error
```

Comment:
I think this should clearly be a "when comitting a deleted key ownership, throw an error"

Reply:
Updated to `deleting a key ownership throws on commit`